cp-ksql-server/include/etc/confluent/docker/admin.properties.template

```diff
@@ -0,0 +1,4 @@
+{% set security_props = env_to_props('KSQL_', '') -%}
```

Comment:
Why are we populating both `admin.properties.template` as well as `ksql-server.properties.template` with the `KSQL_*` env variables, and with the same variables even? It looks as if the `admin.properties` file is only used for a readiness check in case KSQL should connect to a secure Kafka cluster?

Reply:
This is copied over from the other PR, so those comments still need to be addressed. It's still WIP, so hold off the review until then!